tools/MDTraj/trajectory_converter.xml

```diff
@@ -0,0 +1,61 @@
+<tool id="mdtraj_traj_converter" name="Trajectory Converter" version="1.9.1">
+    <description>convert gromacs trr and gro formats to dcd and pdb formats</description>
+    <requirements>
+        <requirement type="package" version="1.9.1">mdtraj</requirement>
+        <requirement type="package" version="2018.2">gromacs</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+      ln -s '$gro_input' ./groinp.gro &&
+      ln -s '$trr_input' ./trrinp.trr &&
+      mdconvert -o ./dcdout.dcd ./trrinp.trr &&
+      cp ./dcdout.dcd '$dcdout' &&
+      gmx editconf  -f ./groinp.gro -o ./pdbout.pdb &&
+      cp ./pdbout.pdb '$pdbout'
+    ]]></command>
+    <inputs>
+        <param argument="gro_input" type="data" format='gro' label="GRO structure file."/>
+	<param argument="trr_input" type="data" format='trr' label="TRR structure file."/>
+    </inputs>
+    <outputs>
+        <data format="dcd" name="dcdout" label="dcd output"/>
+        <data format="pdb" name="pdbout" label="pdb output"/>
+    </outputs>
+    <tests>
+        <test>
+            <param name="gro_input" value="solv_ions.gro" />
+            <param name="trr_input" value="topol.top" />
```

Comment:
This should be a trr file, not a topology

Reply:
Thanks I will fix this